contrib/postgres_fdw/meson.build

```diff
@@ -20,7 +20,21 @@ postgres_fdw = shared_module('postgres_fdw',
     'dependencies': contrib_mod_args['dependencies'] + [libpq],
   },
 )
-contrib_targets += postgres_fdw
```

Comment:
Why? This time it is completely removed, even if `llvm.found()`

Reply:
Accidentally removed, fixup: https://github.com/percona/postgres/commit/f26737a419783745059b95494dd72e51c309755b